tokio-util/src/udp/frame.rs

```diff
@@ -93,7 +93,8 @@ where
 
                 // Safety: This is guaranteed to be the number of initialized (and read) bytes due
                 // to the invariants provided by `ReadBuf::filled`.
-                unsafe { pin.rd.advance_mut(read.filled().len()) };
+                let filled = read.filled().len();
+                unsafe { pin.rd.advance_mut(filled) };
```

Comment:
Safety comments must be right before the unsafe block. Please move `let filled` before it.